app/views/users/_edit.haml

```diff
@@ -224,7 +228,24 @@
             = link_to t(".request_export_photos"), export_photos_user_path, method: :post,
               class: "btn btn-default"
 
-      .col-md-6
+    .row
+      .col-md-12
+        %h3
+          Import another account into
+          = current_user.diaspora_handle
+        %p
+          diaspora* allows you to merge an account into another, even if the accounts are in two different pods.
```

Comment:
I'll have to come back to this text block. Do remind me.

Reply:
Suggested some modified texts here, for discussion: https://discourse.diasporafoundation.org/t/backup-and-restore-account-migration/544/173